src/test/regress/expected/multi_partitioning.out

```diff
@@ -1255,3 +1255,86 @@ SELECT create_distributed_table('partitioning_test', 'id');
 (1 row)
 
 DROP TABLE partitioning_test;
+-- make sure we can attach partitions to a distributed table in a schema
+CREATE SCHEMA partitioning_schema;
+CREATE TABLE partitioning_schema.schema_test(id int, time date) PARTITION BY RANGE (time);
+SELECT create_distributed_table('partitioning_schema.schema_test', 'id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+CREATE TABLE partitioning_schema.schema_test_2009(id int, time date);
+ALTER TABLE partitioning_schema.schema_test ATTACH PARTITION partitioning_schema.schema_test_2009 FOR VALUES FROM ('2009-01-01') TO ('2010-01-01');
```

Comment:
Would be nice to have one tests for each with `set search_path = ...` and attach partitions